src/ovro_lwa_portal/ingest/prefect_workflow.py

```
@@ -59,6 +60,20 @@ def wrapper(*args, **kwargs):  # type: ignore[no-untyped-def]
+
+    def _execute_conversion_should_retry(
+        _task: object, _task_run: object, state: object
+    ) -> bool:
+        """Return False for duplicate-FITS RuntimeErrors: retries will not help."""
+        try:
+            state.result()  # type: ignore[union-attr]
+        except RuntimeError as exc:
+            if _DUP_FITS_ERR in str(exc):
+                return False
+        except Exception:
+            pass
```

Ruff (0.15.11)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)